packages/ferric-example/package.json

```diff
@@ -1,21 +0,0 @@
-{
-  "name": "@react-native-node-api/ferric-example",
```

Comment:
I don't understand the need to remove this in its entirety? While I do agree having a command to generate a project scaffolding, this package serves a purpose as a point of reference in itself.

Reply:
I was going to try and replace that with tests.

Reply:
If the cli are generating the project, then that's the point of reference.

Reply:
> that's the point of reference.

It's hard to share a link to something which comes out of a CLI.

Reply:
https://github.com/callstackincubator/react-native-node-api/pull/200/commits/138fd49cbc31a82fdb7f58d83ee11ae21b510531

Reply:
It could, but only after generated through the testing.

Reply:
I meant mostly as to keep the whole process cleaner.